working/0546-patterns/patterns-feature-specification.md

```diff
@@ -1595,9 +1634,13 @@ To type check a pattern `p` being matched against a value of type `M`:
 
     [nonnull]: https://github.com/dart-lang/language/blob/master/accepted/2.12/nnbd/feature-specification.md#null-promotion
 
-*   **Constant**: Type check the pattern's value in context type `M`. *The
-    context type comes into play for things like type arguments and
-    int-to-double:*
+*   **Constant**: Nothing to do. When type-checking the constant expression,
```

Comment:
Github reviews suck.  Somewhere above the point where this comment is (but which github does not allow me to comment on), there is a line that starts "It is a compile-time error if C? is not assignable to M's == method parameter type.".  I believe the assignability check here is backwards.  That is, you want to check that `C` is assignable to `T?` where `T` is the type of `M`'s parameter type.

Reply:
Good catch.